### ISSUE TITLE
ssl.PROTOCOL_TLS is deprecated warning

### ISSUE THREAD

**Issue by paxri01** [jarun/googler#425]
FYI:

> googler -d
[DEBUG] googler version 4.3.2
[DEBUG] Python version 3.10.0
[DEBUG] Platform: Linux-5.14.16-301.fc35.x86_64-x86_64-with-glibc2.34
/usr/bin/googler:3461: DeprecationWarning: The distutils package is deprecated and slated for removal in Python 3.12. Use setuptools or check PEP 632 for potential alternatives
  from distutils.version import StrictVersion as Version
[DEBUG] Connecting to new host www.google.com
[DEBUG] Opened socket to 108.177.122.106:443
/usr/bin/googler:1611: DeprecationWarning: ssl.PROTOCOL_TLS is deprecated
  ssl_context = ssl.SSLContext(ssl.PROTOCOL_TLS)
[DEBUG] new_connection completed in 0.062s
Please initiate a query.

> python --version
Python 3.10.0

**Comment by capaldo** [comment]
```
[DEBUG] googler version 4.3.2
[DEBUG] Python version 3.10.0
[DEBUG] Platform: macOS-12.0.1-arm64-arm-64bit
/opt/homebrew/bin/googler:3461: DeprecationWarning: The distutils package is deprecated and slated for removal in Python 3.12. Use setuptools or check PEP 632 for potential alternatives
  from distutils.version import StrictVersion as Version
[DEBUG] Connecting to new host www.google.com
/opt/homebrew/bin/googler:1611: DeprecationWarning: ssl.PROTOCOL_TLS is deprecated
  ssl_context = ssl.SSLContext(ssl.PROTOCOL_TLS)
[DEBUG] new_connection completed in 0.077s
Please initiate a query.
```

I'm also experiencing this error.

**Comment by petergi** [comment]
```
/usr/local/bin/googler:1611: DeprecationWarning: ssl.PROTOCOL_TLS is deprecated
  ssl_context = ssl.SSLContext(ssl.PROTOCOL_TLS)
```

**Comment by jarun** [comment]
Guys, raise a PR if you can fix it.

**Comment by jarun** [comment]
Closed at #426.
